spec.md

```diff
@@ -269,19 +269,45 @@ consist of one JSON object that contains both context and data).
 ## Use Cases
 [WIP]  Use-case examples to help end users understand the value of OpenEventing.
 
-### Inter-Service Communication
+### Normalizing Events Across Services & Platforms
 
-### Inter-Platform Communication
+Major event publishers (e.g. AWS, Microsoft, Google, etc.) all publish events in different formats on their respective platforms.  There are even a few cases where services on the same provider publish events in different formats (e.g. AWS).  This forces event consumers to implement custom logic to read or munge event data across platforms and occasionally across services on a single platform.
 
-### Development & Testing of Event-Driven Architectures
+OpenEvents can offer a single experience for authoring consumers that handle events across all platforms and services.
+
+### Facilitating Integrations Across Services & Platforms
+
+Event data being transported across environments is increasingly common.  However, without a common way of describing events, delivery of events across environments is hindered.  There is no single way of determining where an event came from and where it might be going.  This prevents tooling to facilitate successful event delivery and consumers from knowing what to do with event data.
+
+OpenEvents offers useful metadata which middleware and consumers can rely upon to facilitate successful event delivery and receipt.
+
+### Increasing Portability of Functions-as-a-Service
+
+Functions-as-a-Service (also known as serverless computing) is one of the fastest growing trends in IT and it is entirely event-driven.  However, a primary concern of FaaS is vendor lock-in.  This lock-in is partially caused by differences in function APIs and signatures across providers, but the lock-in is also caused by differences in the format of event data received within functions.
```

Comment:
is "entirely" appropriate here since there could be lots of different triggers?

Reply:
This is more of a question than a comment - are we proposing that with the adoption of OpenEvents there would be no vendor lock-in? The vendor lock-in may not come just from the format of the events.

Reply:
@duglin I've changed "entirely" to "largely".

@schatekar I've designed the original language to not indicate that.  Do you think this language could be improved further to communicate this better?